core/src/main/java/com/ibm/wala/ipa/cha/ClassHierarchy.java

```diff
@@ -706,6 +707,111 @@ private void recursiveStringify(Node n, StringBuilder buffer) {
     }
   }
 
+  /*
+   * Puts ClassHierarchy into a JSON readable variable that maps the class to a list of its
+   * subclasses
+   */
+  public Object toJson() {
+    /*
+     * initialize variables (subclass and dag) to store important values like hashmap of each
+     * iteration and a list of all maps of class to subclasses
+     */
```

Comment:
Within a function you should use single-line comments, e.g.:
```suggestion
     // initialize variables (subclass and dag) to store important values like hashmap of each
     // iteration and a list of all maps of class to subclasses
```
(This is not an absolute rule, but it holds for the comments you've written here)